### ISSUE TITLE
Connection Problems (banned)

### ISSUE THREAD

**Issue by blackjack4494** [Splamy/TS3AudioBot#347]
```
2018-08-07 07:43:56.1197| INFO|Ts3Full.TsFullClient_OnDisconnected Calculating up to required security level: 13
2018-08-07 07:43:56.5879| WARN|Ts3Full.TsFullClient_OnDisconnected Could not connect: connect_failed_banned: the command failed to execute: connection failed, you are banned
2018-08-07 07:43:56.5879| INFO|Bot.Dispose Bot disconnecting.
2018-08-07 07:43:56.5935| WARN|Bot.LoggedUpdateBotStatus Internal error (me==null)
2018-08-07 07:43:56.6000| WARN|Bot.GenerateStatusImage Could not save avatar: custom_error: the command failed to execute: Connection closed
2018-08-07 07:43:57.8939| INFO|Bot.InitializeBot Bot connecting...
2018-08-07 07:43:58.3013| WARN|Ts3Full.TsFullClient_OnDisconnected Could not connect: connect_failed_banned: the command failed to execute: connection failed, you are banned
2018-08-07 07:43:58.3013| INFO|Bot.Dispose Bot disconnecting.
2018-08-07 07:43:58.3013| WARN|Bot.LoggedUpdateBotStatus Internal error (me==null)
2018-08-07 07:43:58.3013| WARN|Bot.GenerateStatusImage Could not save avatar: custom_error: the command failed to execute: Connection closed
2018-08-07 07:43:59.4614| INFO|Core.Dispose TS3AudioBot shutting down.

```

We currently have one bot running on a different machine (nl). I was trying some stuff and tried to deploy another one on another machine (de). No matter what I always got the same message you can see in the logs. I tried both solutions in the faq. Tho I tried resetting the uid first but still didn't work. So I applied the b_client_ignore_bans on every group but still the bot cannot connect. I checked the ban list which was empty. I checked the server logs on the actual vps for teamspeak and iptables and stuff.
I then changed the config file to another domain (your ts splamy btw) and it connected. So I wondered. I pulled the master and developer built both but none of them worked on my ts3server. So I restarted the ts3server and everything is working again. I do not know what happened there.

I know that one of my friends were trying something out as well (he deployed a lot of bots in a short amount of time). I guess that broke something on the ts server.

However
tldr if you are getting the error message connect_failed_banned and the faq solutions won't help -> restart the teamspeak server if you have access to that.

I am just curious if someone else noticed something similar

**Comment by iiAlphaWolf** [comment]
How is this solved? I've got the same problem if I try to connect 3+ bots

**Comment by Bluscream** [comment]
because thats not a ts3audiobot problem, thats why

**Comment by Bluscream** [comment]
If you want to know more about Teamspeaks crappy antiflood system, read http://yat.qa/ressourcen/voice-client-anti-flood/

**Comment by Splamy** [comment]
This issue had no activity over 2 months. You didn't leave any comments or reactions, so I can't know how many other people have a similar problem.

Futhermore I don't have any concrete information to work with, hence why I added the 'discussion' tag to it, maybe someone else could add reproducable steps or a test server for me where this happens.

I don't think that this is exactly a bot problem, but the teamspeak server which has a very weird ip banning mechanism in the background. A few of people already have reported similar behaviour that the server bans certain clients but doesn't show _anything_ in the ban list or similar
